commit 01a36a167441121f91907c9558e9ee63082e382d (refs/changes/56/1156/1)
Author: David Pacheco <dap@joyent.com>
Date:   2016-12-21T11:47:23-08:00 (2 years, 10 months ago)
    
    MANTA-3061 storage zone DNS configuration leads to binder overload in large configs

diff --git a/sapi_manifests/registrar/template b/sapi_manifests/registrar/template
index 8fdfaa2..52136ad 100644
--- a/sapi_manifests/registrar/template
+++ b/sapi_manifests/registrar/template
@@ -2,7 +2,7 @@
 	"registration": {
 		"domain": "{{SERVICE_NAME}}",
 		"type": "host",
-		"ttl": 60,
+		"ttl": {{#REGISTRAR_TTL}}{{REGISTRAR_TTL}}{{/REGISTRAR_TTL}}{{^REGISTRAR_TTL}}1800{{/REGISTRAR_TTL}},
 		"aliases": [
 			"{{MANTA_STORAGE_ID}}"
 		]
